### ISSUE TITLE
Why is accuracy used with regression loss

### ISSUE THREAD

**Issue by Lisandro79** [yuanyuanli85/Stacked_Hourglass_Network_Keras#20]
Hi @yuanyuanli85 ,

Thanks for sharing your implementation of Stacked hourglass in Keras. I have a very basic question. In hg_blocks.py (line 22), the model is compiled with the following parameters:

`model.compile(optimizer=rms, loss=mean_squared_error, metrics=["accuracy"])`

May I ask you why is "Accuracy" used as a metric with a regression loss? If the model is performing a regression on the heatmap, why do we need the accuracy here?

Thanks a lot

Best

**Comment by yuanyuanli85** [comment]
you are right. metric accuracy here is useless. it does not provide useful information here.  it is can be removed.

**Comment by Lisandro79** [comment]
I think I will keep `metrics=['mse']` to keep track of the accuracy at predicting the heatmaps.

Thanks for the fast response.

Cheers
